packages/angular/projects/clr-angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -645,16 +643,15 @@
   .datagrid-table-wrapper {
     display: flex;
     flex: 1 1 auto;
+    min-height: 100%;
   }
 
   .datagrid-table {
     display: flex;
     flex-direction: column;
     flex: 1 1 auto;
     align-content: flex-start;
-    min-height: $clr_baselineRem_3; // Includes header and space for a single (non existing) row
```

Comment:
It seems `min-height: 100%`, which I was suggesting, is not really needed here, that's good. 
We can probably return this safeguard to `$clr_baselineRem_3`?
I really don't know why it was included, and I don't like removing things like that, unless unavoidable. Or unexpected regressions may appear.

I am approving this, as it's not a blocker, just a precaution. If you decide (or know the exact origin of the safeguard) you can either return it or commit like this. Thanks for the fix and the fix of the fix.